.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Tip: It's OK to commit this file here, in your project, as this is your personal IDE configuration. But don't forget to add it to `.gitignore` if you will work with other developers.